src/junitParser.ts

```diff
@@ -70,18 +70,17 @@ async function parseFile(file: string, projectTokenDictionaryStrs: string[]) {
     const data: string = fs.readFileSync(file, 'utf8');
     const parser = new XMLParser({ allowBooleanAttributes: true, ignoreAttributes: false, attributeNamePrefix: '' });
     const report = parser.parse(data) as Partial<JUnitReport>;
-
-    return parseSuite(report, file, projectTokenDictionaryStrs);
+    const testSuites = castArray(report.testsuites?.testsuite);
+    return Promise.all((testSuites as JUnitSuite[]).map(async (testSuite: JUnitSuite) => parseSuite(file, projectTokenDictionaryStrs, testSuite)));
```

Comment:
Shouldn't these types all be infered?

```suggestion
    const testSuites = castArray(report.testsuites?.testsuite || []);
    return Promise.all((testSuites).map(async (testSuite) => parseSuite(file, projectTokenDictionaryStrs, testSuite)));
```